File: Api/WebApi/WebApi/Models/RaspberryPi.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebApi.Models
{
    public class RaspberryPi
    {
        public int ID { get; set; }
        public string Location { get; set; }
        public bool IsActive { get; set; }
        public int ProfileID { get; set; }
        public string Longitude { get; set; }
        public string Latitude { get; set; }
    }
}

File: frontend/src/classes/RaspberryPi.ts
export default class RaspberryPi {
    Id: number;
    Location: string;
    IsActive: boolean;
    ProfileID: number;
    Longitude: number;
    Latitude: number;
    IsAccountConfirmed: boolean;

    constructor(id: number, location: string, isActive: boolean, profileID: number, longitude: number, latitude: number, isAccountConfirmed: boolean) {
        this.Id = id;
        this.Location = location;
        this.IsActive = isActive;
        this.ProfileID = profileID;
        this.Longitude = longitude;
        this.Latitude = latitude;
        this.IsAccountConfirmed = isAccountConfirmed;
    }
}